core/consensus/grandpa/impl/voting_round_impl.cpp

```diff
@@ -381,6 +382,10 @@ namespace kagome::consensus::grandpa {
       }
       switch (state_) {
         case State::START:
+          // if we are primary and in the start state during prevote, then error happened during precommit. Should stop
+          if (isPrimary()) {
+            break;
```

Comment:
maybe add error to log

Reply:
By this point error should be already sent to env_->onCompleted and logged there